firestore/integration_test_internal/src/firestore_test.cc

```diff
@@ -1290,6 +1304,16 @@ TEST_F(FirestoreIntegrationTest, TerminateCanBeCalledMultipleTimes) {
 }
 #endif  // defined(__ANDROID__) && FIRESTORE_HAVE_EXCEPTIONS
 
+TEST_F(FirestoreIntegrationTest, canTerminateFirestoreInstance) {
```

Comment:
```suggestion
TEST_F(FirestoreIntegrationTest, CanTerminateFirestoreInstance) {
```

Reply:
Wow, this "Suggested Change" is amazing.  It allows me to accept the suggestion directly!